Makefile

```
@@ -102,10 +102,9 @@ demo-failover: ## Execute failover demo
 
 # spin-up local environment
 .PHONY: deploy-full-local-setup
-deploy-full-local-setup: ## Deploy full local multicluster setup
-	docker network create --driver=bridge --subnet=172.16.0.0/24 $(CLUSTER_GSLB_NETWORK)
-	$(call create-local-cluster,$(CLUSTER_GSLB1),-p "80:80@agent[0]" -p "443:443@agent[0]" -p "5053:53/udp@agent[0]" )
-	$(call create-local-cluster,$(CLUSTER_GSLB2),-p "81:80@agent[0]" -p "444:443@agent[0]" -p "5054:53/udp@agent[0]" )
+deploy-full-local-setup: ## Deploy full local multicluster setup (req. k3d >= 4.0)
```

```suggestion
deploy-full-local-setup: ## Deploy full local multicluster setup (k3d >= 4.0 required)
```
small suggestion, no real need for `req.` abbreviation

---

Fixed, thanks!